src/run_tribler.bat

```diff
@@ -0,0 +1,2 @@
+set PYTHONPATH=%~dp0tribler-core;%~dp0tribler-common;%~dp0tribler-gui
```

Comment:
This doesn't need `anydex` and `ipv8` on the path, like in https://github.com/Tribler/tribler/blob/devel/src/tribler.sh#L13 ?

Reply:
Yep, these should probably be on PATH

Reply:
It actually worked without it, since in the `__init__.py` file in the `tribler-core` module, we add the `pyipv8` directory to the pythonpath. Anydex did not have to be imported since the market community is disabled by default.

Just to be sure, I've added both of them to the bat file 👍